tests/CMakeLists.txt

```diff
@@ -48,10 +49,10 @@ target_link_libraries(unittests
   ${TESTS_LINK_LIBRAIRIES}
 )
 
-add_test(NAME unittests
-         COMMAND unittests
-         WORKING_DIRECTORY ${CMAKE_CURRENT_LIST_DIR}
-         )
+# add_test(NAME unittests
+#          COMMAND unittests
+#          WORKING_DIRECTORY ${CMAKE_CURRENT_LIST_DIR}
+#          )
```

Comment:
Why is this commented out?

Reply:
Because I don't understand the purpose of this :)
As far as I could see, it just runs the tests again, which was a bit boring when I was testing things. But I can revert ofc, or just remove it if it's actually useless.

Reply:
For now I remove it, I can revert if you prefer.

Reply:
I think I would be more comfortable with that `add_test` still active. That `if` clause below may not always be triggered, and for me at least, I'd rather have the tests run twice than not at all.

Reply:
Just to give more details, if you compare the CI logs before and after this change, you can see there is only one difference: before the change there was 1 more test at the end which was run with the following command: `Test command: /home/runner/work/MorphIO/MorphIO/bin/unittests`, i.e. without a test name given (others are run with commands like `Test command: /home/runner/work/MorphIO/MorphIO/bin/unittests "<test_name>"`). So it just runs all the tests at once after they have been run one by one.

But indeed if one sets `EXTERNAL_CATCH2` to `ON`, this might be needed. But in that case it would make more sense to move it in an `else` statement of the `if` you mentioned, isn't it?

Reply:
Yeah, in an `else` would be most appropriate

Reply:
Ok, done